docs-source/content/userguide/managing-domains/domain-resource.md

```diff
@@ -32,6 +32,7 @@ The following prerequisites must be fulfilled before proceeding with the creatio
 * Create a Kubernetes Namespace for the Domain unless the intention is to use the default namespace.
 * Make sure the WebLogic Server Kubernetes Operator is running and is configured to monitor the namespace.
 * Make sure any resources that the domain resource references are deployed to the same namespace. For example, all domain resources have a `spec.webLogicCredentialsSecret` field that references a Kubernetes Secret containing the `username` and `password` of the WebLogic server administrative account.
+* Make sure a domain resource configuration and its corresponding WebLogic configuration [Meet Kubernetes resource name restrictions]({{< relref "/userguide/managing-domains/_index.md#meet-kubernetes-resource-names-restrictions" >}}).
```

Comment:
Meet -> meet

Reply:
I think Dongbo used the capital "Meet" to align with the cross-referenced section title. That's ok but can be changed if you think it looks too odd.